src/components/hero/Hero.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import './hero.scss';
+
+
+const Hero: React.FC = () => {
+  return (
+    <>
+      <a href="#" className="text">I'm link.</a>
+      <h1>Stay tuned, IT world</h1>
```

Comment:
Remember to keep your actual components as much flexible as possible by passing as props anything that may be configured (in order to make component reusable).